pkg/server/telemetry/middleware.go

```diff
@@ -0,0 +1,79 @@
+package telemetry
+
+import (
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/felixge/httpsnoop"
```

Comment:
is `felixge/httpsnoop` already in go.mod?

Reply:
oh good point